src/Snippets/Docs/RateLimiter.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Threading.RateLimiting;
+using Polly;
+using Polly.RateLimiting;
+
+namespace Snippets.Docs;
+
+internal static class RateLimiter
+{
+    public static void Usage()
+    {
+        #region rate-limiter
+
+        // Create a rate limiter with default options that limit to 1000 concurrent executions and no queue.
```

Comment:
Same here about where the 1000 comes from.

Reply:
Dropped, in follow-up will include links to defaults.